src/sentry/api/endpoints/project_rules.py

```diff
@@ -841,9 +846,16 @@ def post(self, request: Request, project) -> Response:
             find_channel_id_for_rule.apply_async(kwargs=kwargs)
             return Response(uuid_context, status=202)
 
-        created_alert_rule_ui_component = trigger_sentry_app_action_creators_for_issues(
-            kwargs["actions"]
-        )
+        try:
+            created_alert_rule_ui_component = trigger_sentry_app_action_creators_for_issues(
+                kwargs["actions"]
+            )
```

Comment:
Potentially cut out the exception handling in the middle and just hadnle the RpcAlertRuleActionResult

Reply:
I thought about this more, and even if we cut out the middle exception raising we still need to handle the RpcAlertRuleActionResult in _some_ helper function, and if we don't raise an exception the typing for the `trigger_sentry_app_action_creators_for_X` get a little more sus, as they will need to return a Response or str or None depending on the function. This case is what I ran into in the [previous PR](https://github.com/getsentry/sentry/pull/82067)